api/api.go

```diff
@@ -730,28 +730,42 @@ func OrbPromote(ctx context.Context, logger *logger.Logger, namespace string, or
 	return &response.PromoteOrb.OrbPromoteResponse.Orb, err
 }
 
+func orbVersionRef(orb string) string {
```

Comment:
This is a fairly simple function, but it might be worth adding a couple of test cases.